new-site/source/scss/layouts/_footer.scss

```diff
@@ -12,3 +12,55 @@
         }
     }
 }
+
+@include media-breakpoint-down(xs) {
+
+    .footer {
+
+        & ul.list-inline {
+
+            .list-inline-item {
+                display: block;
+            }
+
+        }
+    }
+
+}
+
+.bg-green {
+    background-color: $color-green;
+
+    .nav-link {
+        color: $color-white;
+
+        &:hover {
+            color: $--darker-secondary-color;
+        }
+    }
+}
+
+.bg-blue {
+    background-color: $color-blue;
+
+    .nav-link {
+        color: $color-white;
+
+        &:hover {
+            color: $--darker-primary-color;
+        }
+    }
+}
+
+
+.bg-blue {
```

Comment:
This is duplicated

Reply:
Oops. That's an error. I'll fix that now.